packages/mobile/test/schemas.ts

```diff
@@ -783,7 +784,7 @@ export const v24Schema = {
     ...v23Schema.transactions,
     transactions: [],
   },
-  invite: undefined,
+  invites: undefined,
```

Comment:
We shouldn't change this

Reply:
Good catch, looks like a bad merge from my side